xc7/make/arch_define.cmake

```diff
@@ -23,6 +23,7 @@ function(ADD_XC7_ARCH_DEFINE)
       --place_algorithm bounding_box
       --enable_timing_computations off
       --allow_unrelated_clustering on
+      --round_robin_prepacking on
```

Comment:
Can you add a FIXME / TODO on the round-robin-packing to remove it after XYZ is done?

Reply:
done